src/alert.cpp

```diff
@@ -27,6 +27,8 @@ using namespace std;
 map<uint256, CAlert> mapAlerts;
 CCriticalSection cs_mapAlerts;
 
+static void Notify(const std::string& strMessage, bool fThread);
```

Comment:
Actually, there are various usages of the method Notify throughout the file, not just in the method I added.